### PR TITLE
[1.x][Version] Bump to 1.3.0

### PR DESCRIPTION
**Pull request by kavilla**
### Description
Bump OpenSearch Dashboards from 1.2.0 to 1.3.0.

Signed-off-by: Kawika Avilla <kavilla414@gmail.com>
 
### Issues Resolved
n/a
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
    - [x] `yarn test:jest`
    - [x] `yarn test:jest_integration`
    - [x] `yarn test:ftr`
- [ ] ~~New functionality has been documented.~~
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by kavilla** [comment]
Another one to open to talk about. OpenSearch's `1.x` branch is on `1.3.0` so this follows the same version strategy..

**Comment by kavilla** [comment]
Screen Shot 2021-11-23 at 3 48 15 PM

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "1.2.0",
-  "branch": "main",
+  "version": "1.3.0",
+  "branch": "1.2",
```
Comment (kavilla) [inline package.json:15]: Made a proposal [here](https://github.com/opensearch-project/documentation-website/issues/296)

**Review by zuochengding — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (ananzh) [reply package.json:15]: LGTM thanks

**Review by ananzh — APPROVED** [review]

**Pull request merged**